x/commitment/client/wasm/query_staked_positions.go

```diff
@@ -56,6 +56,12 @@ func (oq *Querier) BuildStakedPositionResponseCW(ctx sdk.Context, validators []s
 		tokens := validator.TokensFromSharesTruncated(shares)
 		delegatedAmt := tokens.TruncateInt()
 		votingPower := sdk.NewDecFromInt(validator.Tokens).QuoInt(totalBonded).MulInt(sdk.NewInt(100))
+
+		website := validator.Description.Website
+		if len(website) < 1 {
+			website = " "
```

Comment:
Can we probably remove website empty check and let it be fine with empty website rather than putting space? I think it's not a good approach to bypass the check with `" "`

Reply:
@jelysn Yeah this is the issue that I have updated in the last meetings. JSON marshaller removes the empty value item if it has "omitempty" flag set. Proto3 compiler adds "omitempty" flag by default unless we haven't set nullable=false flag in protobuf declaration. But we can't add the flag for the standard data type like string, int64 etc. You can remember last time I had to use sdk.Int instead of uin64. The main reason was with the omitempty flag in pg.go file generation.